### ISSUE TITLE
Warnings with "flatly" theme

### ISSUE THREAD

**Issue by aeslaughter** [mkdocs/mkdocs-bootswatch#17]
```
Building page documentation/systems/framework/UserObjects/ElementIntegralMaterialProperty.md
WARNING: Template variable warning: 'page_title' is being deprecated and will not be available in a future version. Use 'page.page_title' instead.
WARNING: Template variable warning: 'canonical_url' is being deprecated and will not be available in a future version. Use 'page.canonical_url' instead.
WARNING: Template variable warning: 'next_page' is being deprecated and will not be available in a future version. Use 'page.next_page' instead.
WARNING: Template variable warning: 'previous_page' is being deprecated and will not be available in a future version. Use 'page.previous_page' instead.
WARNING: Template variable warning: 'toc' is being deprecated and will not be available in a future version. Use 'page.toc' instead.
WARNING: Template variable warning: 'content' is being deprecated and will not be available in a future version. Use 'page.content' instead.
WARNING: Template variable warning: 'meta' is being deprecated and will not be available in a future version. Use 'page.meta' instead.
```

FYI. I have a custom logging formater, so it likely looks different than the default verbose output.

**Comment by d0ugal** [comment]
You must be using a development version of MkDocs. This is due to upcoming changes in MkDocs. The bootswatch themes are currently lacking a maintainer, these warnings can be ignored and it will continue to work for some time, until MkDocs 1.0 lands but that currently has no timeline.

Hopefully somebody with enough time or motivation will be able to update the themes at some point.

**Comment by jimporter** [comment]
@d0ugal I can take a look at this. Is there a commit I should reference in MkDocs? I assume most of the changes can just be copied over from the MkDocs theme.

**Comment by d0ugal** [comment]
@jimporter This is the commit: https://github.com/mkdocs/mkdocs/commit/f8a0ac6e03517351c6254a03301996a7b77ab2ca#diff-6a7c9a590efa5c4589e669f54ad36bc9

It's a fairly large one, but if you just look at the `mkdocs/themes/mkdocs/*` files, it is much smaller.

**Comment by d0ugal** [comment]
Also, that would be super! Thank you.

**Comment by memeplex** [comment]
My proposed changed in #21 automatically fixes this, since they sync bootswatch with the upstream mkdocs theme.

**Comment by jimporter** [comment]
I think this is fixed now.

**Comment by waylan** [comment]
This was fixed with #36 (I think is was that one, anyway its been fixed)
